index.html

```diff
@@ -0,0 +1,168 @@
+        <p class="description">
+          World Climate Summit -
+          The Investment COP will utilise a decade of experience to bring together
+          high-level representatives from the public and private sectors including
+          businesses, investors, governments, policymakers, innovators, academia and environmental
+          <influencers class=""></influencers>
+        </p>
+        <h3>7th - 18th November, 2022</h3>
+        <p class="location">Sharm El-Sheikh, Egypt</p>
+      </div>
+    </section>
+
+    <!-- Main Program Events -->
+    <section id="program">
+      <h2 class="section-heading">Main Event</h2>
+      <div class="items">
+        <div class="item">
+          <img src="img/icons-1.svg" class="item-icon" alt="" />
+          <h3 class="item-title">Lecture</h3>
+          <p class="item-description">
+            Lecture to address the climate emergency by challenging existing practices and
+            exploring new ideas to drive tangible results
+          </p>
+        </div>
+        <div class="item">
+          <img src="img/icons-5.svg" class="item-icon" alt="" />
+          <h3 class="item-title">Interaction</h3>
+          <p class="item-description">
+            Both events bring together thought-leaders, industry experts and dynamic organisations
+          </p>
+        </div>
+        <div class="item">
+          <img src="img/icons-3.svg" class="item-icon" alt="" />
+          <h3 class="item-title">Awards</h3>
+          <p class="item-description">
+            Awards
+          </p>
+        </div>
+        <div class="item">
+          <img src="img/icons-4.svg" class="item-icon" alt="" />
+          <h3 class="item-title">Forum</h3>
+          <p class="item-description">
+            Have a chance to share your thoughts and ideas to profer a way
+            forward
+          </p>
+        </div>
+      </div>
+      <a href="#" class="cta">Participate</a>
+    </section>
+
+    <!-- Event Speakers -->
+    <section id="speakers" class="container">
+      <div class="wrapper">
+        <h2 class="section-heading">Speakers</h2>
+
+        <div class="persons"></div>
+
+        <a href="#" id="toggle-more-speakers" class="more">
+          More
+          <svg
+            class="w-6 h-6"
+            fill="none"
+            stroke="currentColor"
+            viewBox="0 0 24 24"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              stroke-linecap="round"
+              stroke-linejoin="round"
+              stroke-width="2"
+              d="M19 9l-7 7-7-7"
+            ></path>
+          </svg>
+        </a>
+      </div>
+    </section>
+
+    <!-- Sponsors and Partners -->
+    <section id="sponsors">
+      <h2 class="section-heading">Partners</h2>
+      <div class="sponsors-list">
+        <img
+          src="img/climate-partners.png"
+          title="sabmiller"
+          alt="sabmiller"
+        />
+      </div>
+    </section>
+
```

Comment:
- [ ] Great design. please kindly use two photos as used in the original design.
yours
![Screenshot (545)](https://user-images.githubusercontent.com/65298048/197073564-5fa02f6c-e3da-40c2-80ef-f4a2026e3a20.png)
behance
![Screenshot (546)](https://user-images.githubusercontent.com/65298048/197073667-cd6fb7e0-1b46-490d-80cb-397063c84890.png)